HeterogeneousCore/AlpakaInterface/interface/TransferToHost.h

```diff
@@ -0,0 +1,21 @@
+#ifndef HeterogeneousCore_AlpakaInterface_interface_TransferToHost_h
+#define HeterogeneousCore_AlpakaInterface_interface_TransferToHost_h
+
+// TODO: better package?
```

Comment:
This file is one of those that should not be in a package depending on `FWCore/Framework` (but in one that fulfills the DataFormat constraints), but is specific to CMS so would not really fit in `HeterogeneousCore/AlpakaInterface`.